lib/rules/no-replace-test-comments.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @fileoverview No 'Replace this with your real tests' comments
+ * @author Jay Gruber
+ */
+
+'use strict';
+
+const ERROR_MESSAGE = "No 'Replace this with your real tests' comments. Add a substantive test.";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: "disallow 'Replace this with your real tests' comments",
+      category: 'Testing',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-replace-test-comments.md',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  ERROR_MESSAGE,
+
+  create(context) {
```

Comment:
We have a util `emberUtils.isTestFile` that can be used to ensure this rule only runs on test files.